### ISSUE TITLE
Ensure NVDA can tab through cookie msg elements

### ISSUE THREAD

**Issue by AndyEPhipps** [comicrelief/storybook#247]
Seems like, possibly because it's being loaded after the DOM has built, NVDA tabbing is missing the button and link (reported in https://github.com/comicrelief/giftaid-react/pull/81)
